backend/endpoints/api.py

```diff
@@ -11,10 +13,20 @@
 
 
 @api.exception_handler(AuthenticationError)
-def authentication_error(request, e):
+def authentication_error(request: HttpRequest, e: AuthenticationError) -> HttpResponse:
     print(e)
     return api.create_response(
         request,
         {"error": "Could not authenticate. Hint: You need to add 'Authorization: Bearer supersecret;' to your HTTP headers. This can be done in Swagger UI using the lock icon in the top right of the endpoint."},
         status=401
```

Comment:
aren't there any error codes constants we could use instead of hardcoding them?

Reply:
Good point - will look into this. For the sake of argument, changing them to constants would make it harder to figure out when switching between frontend and backend (as constant names will be different).